lmdb-sys/Cargo.toml

```diff
@@ -20,5 +20,4 @@ libc = "0.2"
 
 [build-dependencies]
 pkg-config = "0.3"
-# https://github.com/alexcrichton/gcc-rs/issues/240
-gcc = "=0.3.51"
```

Comment:
Since this is effectively bumping the minimum required rustc version, could you also bump the tested version in .travis.yml in this commit?